packages/docz-tools/README.md

```diff
@@ -87,3 +104,32 @@ export default {
   }
 }
 ```
+
+### Hide / Show Actions
+
+The actions in the top right can be hidden if required by adjusting `hasActions`
+to false. By default it is `true`
+
+```
+// doczrc.js
+export default {
+  ...,
+  themeConfig: {
+    ...,
+    hasActions: false
+  }
+}
+```
+
+### Hide / Show Logo
+
+The logo within the sidebar can be hidden if required by providing an
+`undefined` value to `title`.
+
+```
+// doczrc.js
+export default {
+  ...,
+  title: undefined
+}
+```
```

Comment:
FYI, it is required to pass `undefined` to title if you don't want the logo to display, as docz will grab the name from your `package.json` if it does not find the title within `doczrc.js`.